vochain/transaction/vote_tx.go

```diff
@@ -14,25 +14,35 @@ import (
 
 // VoteTxCheck performs basic checks on a vote transaction.
 func (t *TransactionHandler) VoteTxCheck(vtx *vochaintx.VochainTx, forCommit bool) (*vstate.Vote, error) {
```

Comment:
This is the method where I'd need review. It is a very crucial part of the system and the logic should be double-checked. @mvdan @altergui @lucasmenendez @jordipainan